### PR TITLE
Move cice6 to cice.

### PR DESCRIPTION
**Pull request by fischer-ncar**
Rearrange cice5 and cice6.

Contributors other than yourself, if any:

Fixes: [Github issue #s] And brief description of each issue.

User interface changes?: No

Testing performed (automated tests and/or manual tests):

### CODE REVIEW

**Review by dabail10 — COMMENTED** [review]
Doesn't the CIME tag have to be updated as well?

**Comment by mvertens** [comment]
@dbailey @fischer-ncar - yes the CIME tag must be updated also.

File: Externals.cfg [hunk 1]
```diff
@@ -13,18 +13,18 @@ local_path = components/cam
 externals = Externals_CAM.cfg
 required = True
 
-[cice]
+[cice5]
 tag = cice5_20210802
```
[the remark this answers is not in the document]
Comment (fischer-ncar) [reply Externals.cfg]: Thanks, I missed that.

File: Externals.cfg [hunk 2]
```diff
@@ -13,18 +13,18 @@ local_path = components/cam
...
 protocol = git
 repo_url = https://github.com/ESCOMP/CESM_CICE5
-local_path = components/cice
+local_path = components/cice5
 required = True
 
 [cice6]
 tag = cesm_cice6_2_0_5
```
[the remark this answers is not in the document]
Comment (fischer-ncar) [reply Externals.cfg]: Thanks, missed that.

**Comment by fischer-ncar** [comment]
Yes, I did update the cime tag in my normal workflow.  I normally just push the new Externals directly to master.  
But since I wasn't 100% about where cice5 and cice6 are going I wanted to make it a PR.  But I'll go ahead and
update the cime tag here.

**Pull request merged**